treeherder/config/settings.py

```diff
@@ -114,7 +114,7 @@
 #
 # which django-environ converts into the Django DB settings dict format.
 DATABASES = {
-    'default': env.db_url('DATABASE_URL'),
+    'default': env.db_url('DATABASE_URL', default='http://root@mysql/treeherder'),
```

Comment:
Reminder @armenzg 
This is a comment for my own self: Keep an eye on this differing value. This is probably http://root@localhost:3306/treeherder

Reply:
HI @shubhank-saxena remove 'http'. That is why you get an issue.

Reply:
Removed! That issue is fixed. Now getting new issue! Let me try to correct it

Reply:
I still see "http" in the diff. Did you forget to push your latest changes?

Reply:
Yes sir. I have tried it locally. Will push it as soon as other issue is solved

Reply:
Change it to this:
`root@localhost:3306/treeherder`

This represents `user@host:port/database`.